start_esp/start_esp.py

```diff
@@ -569,6 +571,14 @@ def make_argparser():
         action='store_true',
         help=argparse.SUPPRESS)
 
+    parser.add_argument('--client_ip_header', default=None, help='''
+    Defines the request header field whose value will be used to be used as
```

Comment:
Defines the HTTP header name to extract client IP address.

Reply:
Done